how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,135 @@
+Then one day I attended a meetup where some guy enthusiastically demonstrated AngularJS. He made a list of hospital employees which could be clicked to unfold a list of their certifications and skills. A very simple page. And the amount of code that was shipped to make this simple page work was over 1mb.
+
+So that was it for me. This new stuff was overly complex, scary and most of all: I could write this accordion list in less than 1kb of vanilla JS. Or maybe even without JS in CSS and HTML only… My common sense said: "Why the **** would you use something _so_ complex for something _so_ simple?!"
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_wtf1.png)  
+
+At that time I was the only frontend developer at [MyOnlineStore](https://www.myonlinestore.com/). And I have always been the only frontend developer for all my previous employers. So there was nobody to convince me that it would be wise to embrace these new techniques.
+
+## Why I stood still for so long
+
+In late 2015, when I was already late to the party I got sick. A severe burnout followed by a big relapse one year later while I was still recovering. When I felt a bit better after a while and came back to the [MyOnlineStore](https://www.myonlinestore.com/) offices everything had changed. There were two new frontend developers. With new repos, new code, new words, new everything!
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_wtf2.png)
+  
+
+It was 2017 already. I did not know where to start. I didn't dare to ask for help, because of my misplaced pride. I was writing code before some of my new colleagues were even born… and now I should go to these guys and admit I had no clue what I was doing?
+
+But I tried… I tried to keep up with it for a few months and then I was like: "I don't get it! fuck this shit!" 
+
+"I have been doing design _and_ development 50/50 for all of my career, so now it's time to say goodbye to one of these and focus on the other." And since I got _so_ scared of The New Frontend World, I fled to the design department of [MyOnlineStore](https://www.myonlinestore.com/). This was in early 2018.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan-drawing.gif)
+
+_Yes, this is me doing design_
+
+Unfortunately, after a year of doing (UX) design, I found out this was the wrong choice. After years of intensive therapy and running straight into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I am able to design, but I can't handle the stress that goes with it. I felt like I had to answer and fight for all the design choices I made. And since my UX designing is mostly driven by instinct, this was really hard for me. And on top of that it all felt very personal. Maybe I can handle it today or in the near future, with all the things I have learned since, but not then. It fueled my insecurity enormously.
```

Comment:
```suggestion
Unfortunately, after a year of doing (UX) design, I found out this was the wrong choice. After years of intensive therapy and running straight into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I am able to design, but I can't handle the stress that goes with it. I felt like I had to answer and fight for all the design choices I made. And since my UX designing was mostly driven by instinct, this was really hard for me. On top of that it all felt very personal. Maybe I can handle it today or in the near future, with all the things I have learned since, but not then. It fueled my insecurity enormously.
```